tools/Dockerfile

```diff
@@ -53,8 +53,8 @@ RUN apt-get update -qq \
   vim \
   nano \
   groff \
-  awscli \
   rsync \
+  postgresql-client \
```

Comment:
Would it be possible to use `bitnami/postgresql` that contains all required tooling?

Reply:
Ah I see it is noot that easy as the aws CLI is also needed